caluma/caluma_form/tests/test_validators.py

```diff
@@ -126,60 +126,6 @@ def test_validate_dynamic_option_exists(
     assert DocumentValidator().validate(dynamic_option.document, info) is None
 
 
-@pytest.mark.parametrize(
-    "required_jexl,should_throw",
-    [("true", True), ("false", False), ("form == 'main'", True)],
-)
-def test_validate_nested_form(
-    db,
-    required_jexl,
-    should_throw,
-    form_question_factory,
-    document_factory,
-    answer_factory,
-    info,
-):
-    sub_form_question_1 = form_question_factory(
-        form__slug="sub_1",
-        question__type=Question.TYPE_TEXT,
-        question__slug="sub_1_question_1",
-    )
-    sub_form_question_2 = form_question_factory(
-        form__slug="sub_2",
-        question__type=Question.TYPE_TEXT,
-        question__is_required=required_jexl,
-        question__slug="sub_2_question_1",
-    )
-
-    main_form_question_1 = form_question_factory(
-        form__slug="main",
-        question__type=Question.TYPE_FORM,
-        question__sub_form=sub_form_question_1.form,
-        question__slug="sub_1",
-        question__is_required="false",
-    )
-    form_question_factory(
-        form=main_form_question_1.form,
-        question__type=Question.TYPE_FORM,
-        question__sub_form=sub_form_question_2.form,
-        question__slug="sub_2",
-        question__is_required="false",
-    )
-
-    main_document = document_factory(form=main_form_question_1.form)
-
-    answer_factory(
-        document=main_document, question=sub_form_question_1.question, value="foo"
-    )
-
-    if should_throw:
-        error_msg = f"Questions {sub_form_question_2.question.slug} are required but not provided"
-        with pytest.raises(ValidationError, match=error_msg):
-            DocumentValidator().validate(main_document, info)
-    else:
-        DocumentValidator().validate(main_document, info)
-
-
```

Comment:
Why delete the test?

Reply:
The test is completely covered by `test_validate_hidden_subform`, so no need to have two tests doing the same thing